engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/sink/AvroSinkValueParameterTest.scala

```diff
@@ -0,0 +1,38 @@
+package pl.touk.nussknacker.engine.avro.sink
+
+import org.scalatest.{FunSuite, Matchers}
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.api.definition.{DualParameterEditor, Parameter, StringParameterEditor}
+import pl.touk.nussknacker.engine.api.editor.DualEditorMode
+import pl.touk.nussknacker.engine.api.typed.typing
+import pl.touk.nussknacker.engine.avro.KafkaAvroBaseTransformer.SinkValueParamName
+import pl.touk.nussknacker.engine.util.typing.TypingUtils
+
+class AvroSinkValueParameterTest extends FunSuite with Matchers {
```

Comment:
Can you add some tests also for error cases?

Reply:
:+1: 
https://github.com/TouK/nussknacker/pull/1405/commits/c502a5f0ed1fae5369bcdacfa85253d45f6f74b6